### DISCUSSION TITLE
Multiple checks before autocomplete starts

### DISCUSSION THREAD

**Discussion by alanski66** [TarekRaafat/autoComplete.js#293] [Q&A] [answered]
I want to check if the input value is a postcode before autocomplete starts.
Is there anywhere in the API /config that lets me wait for 3 characters to be entered bind a listener and then decide to use autocomplete data src or not?

**Comment by folknor** [comment, marked as answer]
Yes, you can for example use https://tarekraafat.github.io/autoComplete.js/#/configuration?id=trigger-optional which accepts a function. Just return `true` if you want the autoComplete.js engine to proceed and any falsy value if not.

**Reply by alanski66** [reply]
Awesome thank you
